src/src/pages/capabilities/details.js

```diff
@@ -45,6 +47,18 @@ function CapabilityDetailsPageContent() {
 
   const pagetitle = isDeleted ? `${name} [Deleted]` : name;
 
+  const [showJsonMetadata, setShowJsonMetadata] = useState(false);
+  useEffect(() => {
+    if (
+      links &&
+      links.metadata &&
+      links.metadata.allow.includes("PUT") &&
+      links.metadata.allow.includes("GET")
```

Comment:
Maybe these four lines are more readable as  either
```
(links.metadata.allow || []).includes("GET") &&
(links.metadata.allow || []).includes("PUT")
```
or
```
(links?.metadata?.allow?.includes("GET") &&
(links?.metadata?.allow?.includes("PUT")
```

Reply:
changed to:
```
(links?.metadata?.allow || []).includes("GET") &&
(links?.metadata?.allow || []).includes("PUT")
```